From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Daniel Bradley <daniel@pulumi.com>
Date: Wed, 22 May 2024 12:49:00 +0100
Subject: [PATCH] Relax storage management policy validation

Remove validation of tier_to_cool_after_days_since_last_access_time_greater_than range.

diff --git a/internal/services/storage/storage_management_policy_resource.go b/internal/services/storage/storage_management_policy_resource.go
index 244d92f98c..45588bd001 100644
--- a/internal/services/storage/storage_management_policy_resource.go
+++ b/internal/services/storage/storage_management_policy_resource.go
@@ -139,10 +139,9 @@ func resourceStorageManagementPolicy() *pluginsdk.Resource {
 													ValidateFunc: validation.IntBetween(0, 99999),
 												},
 												"tier_to_cool_after_days_since_last_access_time_greater_than": {
-													Type:         pluginsdk.TypeInt,
-													Optional:     true,
-													Default:      -1,
-													ValidateFunc: validation.IntBetween(0, 99999),
+													Type:     pluginsdk.TypeInt,
+													Optional: true,
+													Default:  -1,
 												},
 												"auto_tier_to_hot_from_cool_enabled": {
 													Type:     pluginsdk.TypeBool,
